src/controller/tests/Test_cert_data.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+ *
+ *    Copyright (c) 2020-2025 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      This file contains definitions of test certificates used by unit tests.
+ *
+ */
+
+#include "Test_cert_data.h"
+#include <lib/support/CodeUtils.h>
+#include <lib/support/Span.h>
+// certificates are DER encoded
+
+namespace chip {
+namespace TestCerts {
+
+constexpr uint8_t kTestRCAC[] = {
```

Comment:
Need some comments on what these represent or how we got them. What are these? Are they random (and then why not some `kTestRCAC[] = {1,2,3}` ) or real. I believe they are real and then I would want to know how they were generated/where they originated and what makes them "correct".